modules/eventsource/event.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package eventsource
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+)
+
+func wrapNewlines(w io.Writer, prefix []byte, value []byte) (sum int64, err error) {
+	if len(value) == 0 {
+		return
+	}
+	n := 0
```

Comment:
```suggestion
	n := int64(0)
```
And you can skip the conversion on `sum += int64(n)`.

Reply:
Nope.

```go
type Writer interface {
    Write(p []byte) (n int, err error)
}
```

Means n has to be an `int` not an `int64`